# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ImageContainer.cpp
# Commit: bb0a2ae7c996
# Full Hash: bb0a2ae7c99684152f29b22ba1aa31b3ae1beb5e
# Author: Kelsey Gilbert <jgilbert@mozilla.com>
# Date: 2021-12-16 16:31:55
# Description:
#   Bug 1746135 - Fix a crash on (some) tier2 Android. r=gfx-reviewers,sotaro
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D133972
# ==============================================================================

diff -r b4848e53e40c -r bb0a2ae7c996 gfx/layers/ImageContainer.cpp
--- a/gfx/layers/ImageContainer.cpp	Thu Dec 16 07:48:37 2021 +0000
+++ b/gfx/layers/ImageContainer.cpp	Thu Dec 16 08:01:42 2021 +0000
@@ -222,6 +222,9 @@
   if (!tcOverride) {
     tc = GetTextureClient(nullptr);
   }
+  if (!tc) {
+    return {};
+  }
 
   const auto& tcd = tc->GetInternalData();
 
